doc/UserGuide.md

```diff
@@ -53,16 +54,20 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorts all persons : `sort`
+Sorts all persons in the address book by name only. Ignores case. Relative order of persons with same name is preserved<br>
```

Comment:
Perhaps you can elaborate if the name is sorted in ascending or descending order =)?